src/main/webapp/js/query.js

```diff
@@ -66,7 +47,10 @@ function reroll() {
                 pickEl.innerText = response.pick.name;
                 ratingEl.innerText = response.pick.rating + ' ★';
                 let photoUrl = 'https://maps.googleapis.com/maps/api/place/photo?maxwidth=600&photoreference=' + response.pick.photos[0].photoReference + '&key=AIzaSyBL_9GfCUu7DGDvHdtlM8CaAywE2bVFVJc';
+                localStorage.setItem("restaurantAddress", response.pick.vicinity);
                 loadImage(photoUrl);
+                // addMapScript(); // JANK ALERT: don't add maps script multiple times; fix this in the future
```

Comment:
Please use better language in your comments & don't leave in commented out code. With a few exceptions, TODOs generally should be descriptive enough without leaving commented out code

Reply:
So is the screenshot attached to this PR with the `addMapScript();` not commented out?

Reply:
Sorry, this was a comment for myself to remind me to remove this statement once done with barba.js testing. addMapsScript is run in a different section, so this line is unnecessary.